crates/tx-parser/src/deposit/test_utils.rs

```diff
@@ -22,12 +22,14 @@ pub fn test_taproot_addr() -> BitcoinAddress {
     BitcoinAddress::parse(&addr.to_string(), bitcoin::Network::Regtest).unwrap()
```

Comment:
```suggestion
    BitcoinAddress::parse(&addr.to_string(), Network::Regtest).unwrap()
```
can we move this import up.